models/chainlink/CCIP/chainlink_CCIP_events.sql

```diff
@@ -0,0 +1,123 @@
+{{ config(
+        tags=['dunesql'],
+        alias = alias('CCIP_events'),
```

Comment:
We may use the strict lower_snake_case for the alias:
```suggestion
        alias = alias('ccip_events'),
```